src/Executable.js

```diff
@@ -193,7 +195,8 @@ export default class Executable {
         return (
             error.status === GrpcStatus.Unavailable ||
             error.status === GrpcStatus.ResourceExhausted ||
-            error.status === GrpcStatus.Internal
+            (error.status === GrpcStatus.Internal &&
+                RST_STREAM.test(error.message))
```

Comment:
Should have a unit test that at least tests the retry handler with various errors to ensure it produces the desired boolean.